modin/experimental/cloud/rayscale.py

```diff
@@ -141,18 +141,25 @@ def __make_config(self):
 
         return _bootstrap_config(config)
 
-    @classmethod
-    def _conda_requirements(cls):
+    def _conda_requirements(self):
+        import shlex
+
         reqs = []
 
-        reqs.append(f"python=={cls._get_python_version()}")
+        reqs.append(f"python=={self._get_python_version()}")
+
+        if self.add_conda_packages:
+            reqs.extend(self.add_conda_packages)
+
+        # this is needed, for example, for dependencies that
+        # looks like: "scikit-learn>=0.23"
+        reqs_with_quotes = [shlex.quote(req) for req in reqs]
```

Comment:
lol, this isn't what I was saying :) look (here and below):
```suggestion
        return reqs
```